cmd/tendermint/commands/lite.go

```diff
@@ -59,6 +59,11 @@ func ensureAddrHasSchemeOrDefaultToTCP(addr string) (string, error) {
 }
 
 func runProxy(cmd *cobra.Command, args []string) error {
+	// Stop upon receiving SIGTERM or CTRL-C.
+	cmn.TrapSignal(logger, func() {
+		// TODO: close up shop
```

Comment:
Is there anything of significant importance to do here?

Reply:
stop the server? created https://github.com/tendermint/tendermint/issues/3318